src/tidy_code.py

```diff
@@ -0,0 +1,38 @@
+from random import randint as a
```

Comment:
> Try to understand what the Python program below does. Then copy the code into file tidy_code.py, and make it comprehensible by changing variable and function names, and maybe also reorganise code.

I can see that you understand what the code does but the second part of the question is not attempted, sadly i cannot accept this.